### PR TITLE
Add bootstrap file to avoid duplicated code.

### PR DESCRIPTION
**Pull request by Lutacon**
Update console and index files to the new version and avoid duplicated code.

### CODE REVIEW

File: bin/console [hunk 1]
```diff
@@ -5,35 +5,34 @@ use App\Infrastructure\Kernel;
...
-use Symfony\Component\Dotenv\Dotenv;
 
 set_time_limit(0);
 
-require __DIR__.'/../vendor/autoload.php';
+require dirname(__DIR__).'/vendor/autoload.php';
 
 if (!class_exists(Application::class)) {
-    throw new \RuntimeException('You need to add "symfony/framework-bundle" as a Composer dependency.');
+    throw new RuntimeException('You need to add "symfony/framework-bundle" as a Composer dependency.');
```
Comment (jorge07) [inline bin/console]: use `\RuntimeException` please

File: config/bootstrap.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require dirname(__DIR__) . '/vendor/autoload.php';
+
+// Load cached env vars if the .env.local.php file exists
+// Run "composer dump-env prod" to create it (requires symfony/flex >=1.2)
+if (is_array($env = @include dirname(__DIR__) . '/.env.local.php')) {
```
Comment (jorge07) [inline config/bootstrap.php]: `@include` hurst me. Can we avoid this?

File: bin/console (hunk 1 above)
Comment (Lutacon) [reply bin/console]: Is necessary the qualifier? 🤔

File: config/bootstrap.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require dirname(__DIR__) . '/vendor/autoload.php';
+
+// Load cached env vars if the .env.local.php file exists
+// Run "composer dump-env prod" to create it (requires symfony/flex >=1.2)
```
Comment (jorge07) [inline config/bootstrap.php]: If this is needed it should live in the Dockerfile. But... I tried localy and it dumps the env vars into an array file to improve the performance but there're some problems here:
- This cached config file MUST be generated before app start
- Containers are immutable and is the orchestrator who binds the config before run it via volume mounts from secrets or config maps, or via env vars... Following this this command must be called at preStart but, why the need of composer in production? So I spent some time playing with it and I don't like workflow. At all, it's so hacky, and indicates that there's a problem in the workflow. This file can be generated as part of the cache, generated at the first requests and that's it and outside composer. 
I ended opening an issue https://github.com/symfony/flex/issues/512
At the moment, red light.

Comment (jorge07) [reply config/bootstrap.php]: Bugazo https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/4.2/config/bootstrap.php#L11

Comment (Lutacon) [reply config/bootstrap.php]: wtf? O_o

Comment (Lutacon) [reply config/bootstrap.php]: https://github.com/symfony/recipes/commit/65dc172421cd7c70e8e46759c02b69faa41a95b4

Comment (jorge07) [reply config/bootstrap.php]: IMHO this should be use in prod... DotEnv I mean

**Pull request closed without merging**